packages/paste-website/src/components/shortcodes/table-of-contents/styles.ts

```diff
@@ -36,7 +36,7 @@ export const fontSizePartial = ({depth}: TableOfContentsListItemProps): string =
   return themeGet('fontSizes.fontSize30');
 };
 
-export const StyledListItem: React.FC<TableOfContentsListItemProps> = styled.li`
+export const StyledListItem = styled.li<TableOfContentsListItemProps>`
```

Comment:
Derp, so that was the issue huh?

Reply:
Yeah... it was kind of frustrating. We do this in other components. I still think the function component route _should_ work, but what ever, this works for us now.